### PR TITLE
Go test cover

### PR DESCRIPTION
**Pull request by chai2010**
1. add godoc/Coverage/license badges
2. update readme

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+- With issues:
+  - Use the search tool before opening a new issue.
+  - Please provide source code and commit sha if you found a bug.
+  - Review existing issues and provide feedback or react to them.
+
+## Description
```
Comment (ldxdl) [inline .github/ISSUE_TEMPLATE.md]: Need templates for:
+ Report a bug
+ Request a feature
+ Ask a question

Comment (chai2010) [reply .github/ISSUE_TEMPLATE.md]: done

Comment (chai2010) [reply .github/ISSUE_TEMPLATE.md]: moved to `.github` repo

**Review by Peefy — APPROVED** [review]
LGTM

**Review by zong-zhe — APPROVED** [review]
LGTM

**Pull request merged**